### PR TITLE
Illustrator support

### PR DESCRIPTION
**Pull request by roperzh**
OK to merge.

Short review.

Summary of changes:

- Add illustrator support

Note to reviewers:

Test aren't ready yet, but I'm working on them.

Completed checkin tasks:

- [x] Did manual testing of interrelated functionality
- [x] Added measurement instrumentation (Mixpanel, etc.)
- [ ] Wrote an automated test covering new functionality

### CODE REVIEW

File: packages/haiku-sdk-creator/src/figma/index.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+export enum ExporterFormat {
+  Unknown = 'Unknown',
+  Bodymovin = 'Bodymovin',
+  HaikuStatic = 'HaikuStatic',
+}
+
+export interface ExporterRequest {
+  format: ExporterFormat;
+  filename: string;
+}
```
Comment (stristr) [inline packages/haiku-sdk-creator/src/figma/index.ts]: Guessing these two interfaces above were inadvertently added and not meant to be included.

File: test.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  var type = ExportType.SVG;
+  var fileSpec = new File(dest);
+  exportOptions.embedRasterImages = true;
+  exportOptions.embedAllFonts = false;
+  exportOptions.cssProperties = SVGCSSPropertyLocation.PRESENTATIONATTRIBUTES;
+  exportOptions.fontSubsetting = SVGFontSubsetting.None;
+  exportOptions.documentEncoding = SVGDocumentEncoding.UTF8;
+  saveMultipleArtboards = true;
+  app.activeDocument.exportFile( fileSpec, type, exportOptions );
+}
```
Comment (stristr) [inline test.jsx]: Inadvertently committed, not intended to be part of this PR?

**Comment by stristr** [comment]
@roperzh, would you mind rebasing to `origin/master`? LMK if you hit any conflicts.

File: packages/haiku-serialization/src/bll/Illustrator.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    fse.writeFileSync(filePath, exportScript)
+
+    execSync(Illustrator.generateExporterCommand(filePath))
+
+    return true
+  }
+
+  static generateExporterCommand (file) {
+    if (isMac()) {
+      return `open -g -b com.adobe.Illustrator ${file}`
```
Comment (tsortwehttam) [inline packages/haiku-serialization/src/bll/Illustrator.js:105]: Might the `file` string need to be escaped/quoted here?

**Comment by tsortwehttam** [comment]
LGTM.

**Pull request merged**